README.md

```diff
@@ -22,6 +22,19 @@ Please note that supporting the [C Programming
 Language](https://en.wikipedia.org/wiki/C_(programming_language)) is not a goal
 of the current project.
 
+## Supported Platforms
```

Comment:
Should this be `Supported Development Platforms` as we don't want to guarantee our code to just run on below platforms?

Reply:
Good point. Will make that explicit.

Reply:
done.